lib/sycamore/sycamore/transforms/standardizer.py

```diff
@@ -0,0 +1,121 @@
+from abc import ABC, abstractmethod
+
+from sycamore.plan_nodes import Node
+
+from sycamore.data import Document
+from sycamore.transforms.map import Map
+from dateutil import parser
+
+
+class Standardizer(ABC):
```

Comment:
Needs docstring.

Reply:
Needs more detail. What is a "Standardizer" and why is it different than a regular map?

Reply:
ping